tests/test_brats_format.py

```diff
@@ -0,0 +1,41 @@
+from unittest import TestCase
+import os.path as osp
+
+import numpy as np
+
+from datumaro.components.annotation import Mask
+from datumaro.components.dataset import Dataset
+from datumaro.components.environment import Environment
+from datumaro.components.extractor import DatasetItem
+from datumaro.plugins.brats_format import BratsImporter
+from datumaro.util.test_utils import compare_datasets
+
+from .requirements import Requirements, mark_requirement
+
+DUMMY_DATASET_DIR = osp.join(osp.dirname(__file__), 'assets', 'brats_dataset')
+
+class BratsImporterTest(TestCase):
+    @mark_requirement(Requirements.DATUM_616)
+    def test_can_detect(self):
+        detected_formats = Environment().detect_dataset(DUMMY_DATASET_DIR)
+        self.assertEqual([BratsImporter.NAME], detected_formats)
+
+    @mark_requirement(Requirements.DATUM_616)
+    def test_can_import(self):
+        expected_dataset = Dataset.from_iterable([
+            DatasetItem(id='BRATS_001',
+                image=np.ones((1, 5, 2, 3)),
```

Comment:
What do these dimensions mean? I don't think it's a regular image.

Reply:
3d images are stored in the dataset. These images can be represented as a set of 2d images.

Maybe we need to store 2d images to easily convert to other formats. The AC has a parameter for this (multi_frame - allows converting annotation of 3D images as sequence of 2D frames (optional, default False))

Reply:
It is the same as video or PCD then, which also have multiple frames inside. Support for new media types is added in #539. 

Current implementation cannot work with all the other system - transforms, filters, inference etc.

Reply:
I replaced `Image` with `PointCloud`. Did you mean this?

Reply:
No, I meant, you probably need a new media type for this. Because neither 3d images, nor multiframe images do not fit in any existing format. Just build upon that PR and mark this dependent.

Reply:
Added a new media format `MultiframeImage`.

Reply:
Ok, please check if you can contribute to #539